tests/snapshots/src/main/generated/minimized/Enums.java

```diff
@@ -1,4 +1,5 @@
 package minimized;
+^^^^^ reference minimized/Enums#
```

Comment:
This one is unusual :thinking: @olafurpg

Reply:
🤔 Hmmm not sure what's going on here. It's not a blocking issue. I think it makes sense that we try to document these quirks at some later point as issues. For now, it's most important to the the infrastructure working.